meinberlin/apps/dashboard/templates/meinberlin_dashboard/module_blueprint_list_dashboard.html

```diff
@@ -13,3 +15,6 @@ <h1>{% trans "Choose Participation Module" %}</h1>
     </ul>
 </div>
 {% endblock %}
+
+{% block footer %}
+{% endblock %}
```

Comment:
I think it was there to use the stuff from the dashboard? No, I think it can't because that's in the view and the template shouldn't matter. Let's just try it out!